### PR TITLE
Column added to RequestIssues table

### PR DESCRIPTION
**Pull request by buddylove954**
Resolves [Jira Issue Title](https://jira.devops.va.gov/browse/JIRA-12345)

# Description

Added additional column for the request issue table for nonrating_issue_bgs_source

Local testing:
Screen Shot 2024-05-10 at 11 32 12 AM
Screen Shot 2024-05-10 at 10 28 04 AM
Screen Shot 2024-05-10 at 10 27 43 AM

## Acceptance Criteria

- [x] column name is nonrating_issue_bgs_source
- [x] column description is Name of Table in Corporate Database where the nonrating issue is stored.  This datapoint is correlated with the nonrating_issue_bgs_id.
- [x] has a datatype of varchar

## Testing Plan

1. Go to [Test plan 42703 for APPEALS-45147](https://jira.devops.va.gov/browse/APPEALS-45147)
2. Go to [XRAY for APPEALS-45674](https://jira.devops.va.gov/secure/XrayExecuteTest!default.jspa?testExecIssueKey=APPEALS-45677&testIssueKey=APPEALS-45674)

### Test Coverage
Did you include any test coverage for your code? Check below:
- [x] RSpec

### Code Climate
Your code does not add any new code climate offenses? If so why?
- [x] No new code climate issues added

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -623,76 +619,6 @@
     t.index ["updated_by_id"], name: "index_updated_by_id"
   end
 
-  create_table "correspondence_documents", force: :cascade do |t|
```
Comment (jtsangVA) [inline db/schema.rb:626]: Why are all these correspondence tables being removed?

File: db/schema.rb [hunk 2]
```diff
@@ -2108,52 +2028,6 @@
     t.index ["vbms_communication_package_id"], name: "index_vbms_distributions_on_vbms_communication_package_id"
   end
 
-  create_table "vbms_document_types", force: :cascade do |t|
```
Comment (jtsangVA) [inline db/schema.rb:2111]: same here

File: spec/models/request_issue_spec.rb [hunk 3]
```diff
@@ -1089,7 +1090,8 @@
         is_expected.to have_attributes(
           is_predocket_needed: true,
           contested_issue_description: nil,
-          nonrating_issue_description: nil
+          nonrating_issue_description: nil,
+          nonrating_issue_bgs_source: nil
```
Comment (jtsangVA) [inline spec/models/request_issue_spec.rb:1094]: Why is this expect as nil and not "TestSource"?

Comment (jtsangVA) [reply spec/models/request_issue_spec.rb:1094]: can you add a test confirming that assigning a value to that field works?

**Review by jtsangVA — APPROVED** [review]

**Review by nkutub — APPROVED** [review]

**Pull request merged**